onyo/onyo_arguments.py

```diff
@@ -10,36 +10,49 @@
         required=False,
         default=Path.cwd(),
         type=directory,
-        help='Run Onyo commands from inside of DIR'),
+        help="""
+            Run as if Onyo is started in DIR instead of the current working
```

Comment:
Might be nitpicking, but I want to suggest an alternative, because it is not "run as if" but literally run in.

"Run Onyo from DIR instead of the current working directory."

If you consider your version to be more intuitive/understandable feel free to keep it :)